README.md

```diff
@@ -1 +1,20 @@
-# Brewery_Outdoor_Analysis
\ No newline at end of file
+# Brewery_Outdoor_Analysis
+### Topic
+
+Local craft breweries and their correlation to prominant outdoor clothing brands.
+#### Purpose 
+
+Our team believes there is potential for retail growth based on locations in the nearby vicinity of breweries. The increasing popularity of both breweries and outdoor clothing establishments can lead to data that could assist future development.
+
+### Possible questions to answer
+
+- Is there a strong current correlation between breweries and outdoor clothing stores?
+- What is the growth potential for clothing store expansion based on brewery locations?
+- What is the growth potential for brewery expansion based on outdoor clothing locations?
+
+### Collaborators
+
+- Brandon Clark
+- Brandon Driver
+- Tim Nilson
```

Comment:
"Nilsen" hehe -- nobody gets this right. Even my girlfriend had "Nilson"  in her phone when we first started dating 😆